### ISSUE TITLE
Add "GPS_Present" Bit To GPS Bitmask Parsing

### ISSUE THREAD

**Issue by kb1lqd** [FaradayRF/Faraday-Software#56]
**Overview**
An unused bit in the GPS bitmask of the FLASH parsing should be updated to communicate the presence of a GPS unit on the Faraday unit. This will be used to easily determine if a GPS should be onboard or not but does not indicate the status of the GPS unit if installed, for that the presence of the telemetry "GPS FIX" is used.

**Implementation Details**
* Update software to parse "GPS_Present" bit from GPS bitmask
* Update device configuration INI file to include GPS_Present bit configuration
* Update firmware for default unit factory reset value

**Operation**
This bit is used simple to query the presence of a GPS.
* GPS_Present = HIGH
 * GPS is present and Faraday should attempt to parse valid GPS string
* GPS_Present = LOW
 * GPS is a not present and Faraday should default to internally saved default location

**Comment by kb1lqd** [comment]
Post-pone since simply using default location whenever GPS fix not acquired or no GPS string is likely OK.

**Comment by kb1lqd** [comment]
False, need a flash bit since no GPS means no NMEA uart data from unit and thus we'll be stuck without a trigger to trigger the grab from flash or GPS dynamically. Need to add flash bit in and assume it is set correctly.

**Comment by kb1lqd** [comment]
Adding as BIT 1 in GPS Bitmask in flash memory:

![image](https://cloud.githubusercontent.com/assets/339144/22631868/f24c66c2-ebc8-11e6-92ee-b07682aec666.png)

**Comment by kb1lqd** [comment]
GPS Bitmask default in the factory reset can stay 0x01 since default should be no GPS installed and unit configured to have a GPS.

![image](https://cloud.githubusercontent.com/assets/339144/22631895/3c36023e-ebc9-11e6-8a75-9267ca194c81.png)

**Comment by kb1lqd** [comment]
Confirmed addition of bit and operation:

GPS_Present bit HIGH:
![image](https://cloud.githubusercontent.com/assets/339144/22633171/0d5a2b60-ebd4-11e6-9743-c6ac071893c4.png)

GPS_Present bit LOW:
Lon/Lat set to "123456..." along with defaults for all other fields.

![image](https://cloud.githubusercontent.com/assets/339144/22633204/43d8ca84-ebd4-11e6-9fa2-492ee9cb1108.png)

**Comment by kb1lqc** [comment]
Is this implemented @kb1lqd ? Our manual method of telling a unit if it has a GPS installed seems to work fine right now during Faraday programming.

**Comment by el-iso** [comment]
@kb1lqc  Setting the GPS_Present bit worked for me when I was doing setup.

**Comment by kb1lqc** [comment]
Hey @kb1lqd lets chat about this one, I can't tell if it's actually implemented haha...
